README.md

```diff
@@ -5,9 +5,7 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 First, run the development server:
 
 ```bash
-npm run dev
-# or
-yarn dev
+bun dev
```

Comment:
Are we swapping out NodeJS for Bun here?

Reply:
can but not necessary I just use it in development they should be completely interchangeable since for NextJS bun runs node underneath so it's just the improved CLI experience